cmd/api/routers/routers.go

```diff
@@ -33,7 +37,7 @@ func (c *Controller) GetNamespacedResources(context *gin.Context) {
 	group := context.Param("group")
 	version := context.Param("version")
 	namespace := context.Param("namespace")
-	resource := getResourceKindFromPlural(context.Param("resourceType")) + "/" + context.Param("namespace")
+	resource := getAPIResourceKind(context)
```

Comment:
Shouldn't this really be called `kind`?

Reply:
Fixed

Reply:
No, I meant like this:
```
kind := getAPIResourceKind(context)
```

Reply:
see above comment, I think you aren't checking the most updated changes